PhysicsTools/ONNXRuntime/BuildFile.xml

```diff
@@ -1,5 +1,7 @@
 <use name="onnxruntime"/>
 <use name="FWCore/Utilities"/>
+<use name="HeterogeneousCore/CUDAServices" source_only="1"/>
+<use name="FWCore/ServiceRegistry" source_only="1"/>
```

Comment:
`source_only` means the package only depend on the headers of its dependent package and there is no linking required. This flag was not need as headers of dependent packages were always available to include but we added this for  cxxmodules IBs so that scram can properly build the dependent package module first. And no, there is no backward of this supported.

Reply:
ok - thanks - I've updated accordingly..